environments/prod/hearings-reform-hmcts-net.yml

```diff
@@ -0,0 +1,16 @@
+name: "hearings.reform.hmcts.net"
```

Comment:
can you not use the dev.platform.hmcts.net one like the rest of Shared services?

Reply:
I was trying to keep things uniform, so it doesn't confuse the Dev's in the future, please you say what you'd prefer ?

Reply:
I would prefer that we use the same zone for everything if possible, using vh would linking the zone as well for each required env

could you try use dev.platform.hmcts.net please?

Reply:
of course, i'll get it sorted

Reply:
I have removed hearings and used dev.platform